testVectors.json

```diff
@@ -928,9 +937,9 @@
       }
     },
     "keycards": {
-      "0xB8b8985e55aBEa8E36C777c28C08ECBe0104a37d": "0x00297113f51be97cd733d72b3fd7e3a536461d5b",
-      "0xE6bA71C62863DA55baFBCb4EA2099e77C7014601": "0xdd7877dcf849251312ce0d634ac80fed51567875",
-      "0x2E06D5C85Bd38e58E7d7a5FfDdc81552Ed79242f": "0x0000000000000000000000000000000000000000"
+      "0x00297113f51be97cd733d72b3fd7e3a536461d5b": "0xB8b8985e55aBEa8E36C777c28C08ECBe0104a37d",
+      "0xdd7877dcf849251312ce0d634ac80fed51567875": "0xE6bA71C62863DA55baFBCb4EA2099e77C7014601",
+      "0x0000000000000000000000000000000000000000": "0x2E06D5C85Bd38e58E7d7a5FfDdc81552Ed79242f"
     },
     "listings": [
       {
```

Comment:
I'm pretty sure base price for this first listing should be `0x00000000000000000000000000000000000000000000000000000000000001f4`.
I dont think

change_price = mevents.Listing(
```
    id=listing_simple.id,
    base_price=mtypes.Uint256(raw=int(123400).to_bytes(32, "big")),
)
events.append(change_price)
```  

in the gen test vector file is being calculated correctly?